subt/ros/proxy/ros_proxy_node.cc

```diff
@@ -376,8 +379,7 @@ not available.");
              pose.orientation.x, pose.orientation.y, pose.orientation.z, pose.orientation.w);
 
   // Distance to goal
-  double dist = pose.position.x * pose.position.x +
-    pose.position.y * pose.position.y;
+  double dist = sqrt(pose.position.x * pose.position.x + pose.position.y * pose.position.y);
```

Comment:
why do you call `sqrt` here? The comment is misleading, I think it was even taken from original `seed.cc` hello world.

Reply:
How would one know if misleading is the comment + name of the variable or if the code is not correct? Plus output like this is really confusing:

```
1592480426.484447217, 4.780000000  INFO /X0F200L_proxy ros.proxy: MD robot pose -7.42729 5.00544 dist=80.2191
1592480426.836774425, 4.980000000  INFO /X0F200L_proxy ros.proxy: MD robot pose -7.34447 4.9923 dist=78.8644
1592480426.994725326, 5.132000000  INFO /X0F200L_proxy ros.proxy: MD robot pose -7.27034 4.96809 dist=77.5398
1592480427.154557814, 5.280000000  INFO /X0F200L_proxy ros.proxy: MD robot pose -7.19668 4.93235 dist=76.1203
1592480427.256638095, 5.380000000  INFO /X0F200L_proxy ros.proxy: MD robot pose -7.14551 4.90257 dist=75.0935
1592480427.363582254, 5.480000000  INFO /X0F200L_proxy ros.proxy: MD robot pose -7.09334 4.86824 dist=74.0153
1592480427.464359290, 5.580000000  INFO /X0F200L_proxy ros.proxy: MD robot pose -7.04321 4.83155 dist=72.9507
```
Note the "dist" of 80 (meters?).